checkov/main.py

```diff
@@ -574,6 +574,8 @@ def add_parser_args(parser: ArgumentParser) -> None:
                env_var='CKV_SECRETS_SCAN_BLACK_LIST',
                action='append',
                help='black file list to filter out from the secret scanner')
+    parser.add('--summary-position-bottom', action='store_true',
```

Comment:
```suggestion
    parser.add('--summary-position', choices=["top", "bottom"],
```
we could change it to allow two different choices top or bottom and then set the default to top or bottom. of course define them as constants and then they can be evaluated before printing to console.